zhaquirks/tuya/ts000x.py

```diff
@@ -322,6 +322,98 @@ class Switch_2G_Metering(EnchantedDevice, CustomDevice):
     }
 
 
+class Switch_2G_Var03(EnchantedDevice, CustomDevice):
+    """Tuya 2 gang (variation 03))."""
```

Comment:
Second `)` can probably be removed

Reply:
Done: https://github.com/zigpy/zha-device-handlers/pull/2296/commits/0dc12e01f3c47aa4ddc24b6c469e77c0a7b3a9d8